src/yodapa/plugins/asciiator.py

```diff
@@ -0,0 +1,82 @@
+from PIL import Image
```

Comment:
do we need additional pip package for this?

Reply:
yes we need the Pillow package from PIL

`pip install Pillow`